hedera-node/src/test/java/com/hedera/services/txns/schedule/ScheduleCreateTransitionLogicTest.java

```diff
@@ -145,6 +183,252 @@ public void followsHappyPath() {
         verify(txnCtx).setStatus(SUCCESS);
     }
 
+    @Test
+    public void validNotReadyForExecutionOtherPartyKeyNotFound() {
+        // given:
+        givenValidTxnCtx();
+        // and:
+        given(store.getScheduleID(transactionBody, payer)).willReturn(EMPTY_SCHEDULE);
+        given(store.createProvisionally(
+                eq(transactionBody),
+                eq(payer),
+                eq(payer),
+                eq(RichInstant.fromJava(now)),
+                argThat(jKey -> true)))
+                .willReturn(CreationResult.success(schedule));
+        // and:
+        given(store.addSigners(
+                eq(schedule),
+                argThat(jKeySet -> true))).willReturn(OK);
+        // and:
+        given(signingOrder.keysForOtherParties(innerTransactionBody, ScheduleReadyForExecution.PRE_HANDLE_SUMMARY_FACTORY))
+                .willReturn(new SigningOrderResult<>(wrongOtherPartiesKeysList));
+
+        // when:
+        subject.doStateTransition();
+
+        // then:
+        verify(store).getScheduleID(transactionBody, payer);
+        // and:
+        verify(store).createProvisionally(
+                eq(transactionBody),
+                eq(payer),
+                eq(payer),
+                eq(RichInstant.fromJava(now)),
+                argThat((Optional<JKey> k) -> equalUpToDecodability(k.get(), jAdminKey.get())));
+        // and:
+        verify(store).addSigners(
+                eq(schedule),
+                argThat((Set<JKey> set) -> {
+                    assertEquals(set.size(), jKeySet.size());
+                    var setIterator = set.iterator();
+                    var jKeySetIterator = set.iterator();
+                    while (setIterator.hasNext()) {
+                        assertTrue(equalUpToDecodability(setIterator.next(), jKeySetIterator.next()));
+                    }
+                    return true;
+                }));
+        // and:
+        verify(store).commitCreation();
+        verify(txnCtx).setStatus(SUCCESS);
+    }
+
+    @Test
+    public void validNotReadyForExecutionPayerKeyNotFound() {
```

Comment:
Same comment for checking whats going on inside the function. If I comment `readyForExecution` check the test passes too. We need to actually verify the underlying logic. It would be best if we could verify the result of `readyForExecution` method invocation